ucum-java8/pom.xml

```diff
@@ -9,6 +9,18 @@
 	<artifactId>systems-ucum-java8</artifactId>
 	<name>Units of Measurement UCUM System</name>
 	<description>Unit System supporting the Unified Code for Units of Measure (UCUM) on Java SE 8</description>
+	<developers>
+		<developer>
+			<name>Werner Keilw</name>
+			<organization>Creative Arts &#38; Technologies</organization>
```

Comment:
Just a tip: you can also use `&amp;` which might be clearer. but they both mean exactly the same thing.